packages/meditrak-server/src/tests/routes/GETDisasters.test.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { expect } from 'chai';
+import { findOrCreateDummyRecord } from '@tupaia/database';
+import { Authenticator } from '@tupaia/auth';
+import { TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, BES_ADMIN_PERMISSION_GROUP } from '../../permissions';
+import { TestableApp } from '../TestableApp';
+import { prepareStubAndAuthenticate } from './utilities/prepareStubAndAuthenticate';
+
+describe('Permissions checker for GETDisasters', async () => {
+  const DEFAULT_POLICY = {
+    DL: ['Public'],
+    KI: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    SB: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Royal Australasian College of Surgeons'],
+    VU: [TUPAIA_ADMIN_PANEL_PERMISSION_GROUP, 'Admin'],
+    LA: ['Admin'],
+  };
+
+  const BES_ADMIN_POLICY = {
+    SB: [BES_ADMIN_PERMISSION_GROUP],
+  };
+
+  const app = new TestableApp();
+  const { models } = app;
+  let disaster1;
+  let disaster2;
+  let disaster3;
+  let filterString;
+
+  before(async () => {
+    // Set up test disasters in the database
+    disaster1 = await findOrCreateDummyRecord(models.disaster, {
+      name: 'Test disaster 1',
+      countryCode: 'VU',
+    });
+    disaster2 = await findOrCreateDummyRecord(models.disaster, {
+      name: 'Test disaster 2',
+      countryCode: 'KI',
+    });
+    disaster3 = await findOrCreateDummyRecord(models.disaster, {
+      name: 'Test disaster 3',
+      countryCode: 'LA',
+    });
+    filterString = `filter={"id":{"comparator":"in","comparisonValue":["${disaster1.id}", "${disaster2.id}", "${disaster3.id}"]}}`;
+  });
+
+  afterEach(() => {
+    Authenticator.prototype.getAccessPolicyForUser.restore();
+  });
+
+  describe('GET /disasters/:id', async () => {
+    it('Sufficient permissions: Should return a requested disaster that user has access to', async () => {
```

Comment:
Something I've been trying to do more lately is avoid 'Should' in unit test descriptions, it's kind of redundant because every test should do something

Reply:
True, thanks for the reminder! We did discuss this in dev chat a while back too.